### DISCUSSION TITLE
Cluster computing

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [tokio-rs/tokio#6948] [Q&A]
Hi. I'm from a Python server development background and new to Rust and Tokio. I need a cluster computing framework with a master and long-lived workers architecture (eg. a rust equivalent of Dask Distributed in the Python world) which can scale from a multi-core cpu laptop to a cluster of servers. Does Tokio support this?

**Comment by Darksonn** [comment]
That is not what Tokio does. It's like asking "does Python's asyncio library support distributed computing?". The answer is that ... well, if you were to build a distributed computing library for Rust, that library would be very likely to use Tokio. But Tokio itself does not do that.

**Reply by dineshbvadhia** [reply]
@Darksonn Ok. Thanks for the clarification. Do you know if there is a distributed computing library for Rust built atop Tokio (or something else)?
